components/collections/src/codepointtrie/cptrie.rs

```diff
@@ -888,11 +888,11 @@ impl<'trie, T: TrieValue + Into<u32>> CodePointTrie<'trie, T> {
     /// assert_eq!(cp, 0x1158E);
     ///
     /// let plane_num: u8 = trie.get(cp);
-    /// assert_eq!(trie.get_u32(cp), plane_num as u32);
+    /// assert_eq!(trie.get32(cp), plane_num as u32);
     /// ```
     // Note: This API method maintains consistency with the corresponding
     // original ICU APIs.
-    pub fn get_u32(&self, code_point: u32) -> u32 {
+    pub fn get32(&self, code_point: u32) -> u32 {
```

Comment:
Nit: CodePointTrie is weird. In this case, the difference between `get` and `get_u32` is actually the return value; both functions take a u32 argument. I think we should change `get` to `get32`... and name `get_u32` to `get32_u32`?

Reply:
Oh boy that's a weird one. `get32_u32` or `get32_as_u32` seem good

Reply:
`get32_u32` would be consistent :-) Adding an infix to make it read better is fine although less predictable.

Reply:
`get32_u32` would be consistent with `get_ule` (which we're not changing) in the sense that the thing after the `_` is the type of the return value.